integr8ly/tutorial-web-app-operator
# Bump webapp 2.14.0

verification:
- have an openshift 3 cluster
- in the webapp (solution explorer) namespace
  - edit the webapp operator deployment
  - update image to be docker.io/aidenkeatingrht/tutorial-web-app-operator:0.0.24
- wait for the webapp to redeploy
- ensure the webapp image, in the deployment, is 2.14.0
- login to the webapp
- ensure there are two sso instances, one cluster and one standard
- ensure the links both work
